doc/JSON_FLAGS.md

```diff
@@ -780,19 +780,15 @@ Some armor flags, such as `WATCH` and `ALARMCLOCK` are compatible with other ite
 ### Flags
 
 - ```ALWAYS_TWOHAND``` Item is always wielded with two hands. Without this, the items volume and weight are used to calculate this.
-- ```CHOP``` Does cutting damage, with a high chance of getting stuck.
 - ```DIAMOND``` Diamond coating adds 30% bonus to cutting damage
 - ```MESSY``` Resistant to getting stuck in a monster. Potentially cause more gore in the future?
```

Comment:
Needs description change. It does create more gore (especially fun when smashing corpses), but no longer affects sticking.